File: 백준/Bronze/5597. 과제 안 내신 분．．？/과제 안 내신 분．．？.java
import java.util.Scanner;
import java.util.Arrays;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        int[] arr = new int[31];
        
        for(int i = 1; i < 29; i++) {
            int suc = sc.nextInt();
            arr[suc] = 1;
        }
        
        for(int i = 1; i < arr.length; i++) {
            if(arr[i] != 1) {
                System.out.println(i);
            }
        }
    }
}

File: 백준/Bronze/5597. 과제 안 내신 분．．？/과제 안 내신 분．．？.cs
using System;

class Q5597
{
    static void Main(String[] args)
    {
        int[] arr = new int[31];
        
        for(int i = 1; i < 29; i++)
        {
            int suc = int.Parse(Console.ReadLine());
            arr[suc] = 1;
        }
        
        for(int i = 1; i < arr.Length; i++)
        {
            if(arr[i] != 1)
            {
                Console.WriteLine(i);
            }
        }
    }
}